### PR TITLE
Fixing `Globals > ... > Environment` that could not use the `Fn::Ref`

### PR DESCRIPTION
**Pull request by acolombier**
This PR makes this global declarations valid:

```yaml
...
Globals:
    Function:
        Environment:
            Variables:
                GLOB_ENV_VAR: !Ref LocalVar
...
```

Unfortunatelly, I could not find reference that gives details about where can you safely use the `Fn::Ref` function in the `Globals` section, and therefore, some further investigations could come on top of this PR, but the Intrinsic function `Ref` must work for `Environment` section, regarding official AWS Python cfn-lint.

I could not find any relevant test to add.

### CODE REVIEW

File: src/validator.ts [hunk 1]
```diff
@@ -1067,11 +1067,12 @@ function recursiveDecent(ref: any){
             // Check if an Intrinsic function is allowed here
             let inResourceProperty = (placeInTemplate[0] == "Resources" || placeInTemplate[2] == "Properties");
             let inResourceMetadata = (placeInTemplate[0] == "Resources" || placeInTemplate[2] == "Metadata");
+            let inGlobalEnvironment = (placeInTemplate[0] == "Globals" && placeInTemplate[2] == "Environment");
```
Comment (acolombier) [inline src/validator.ts]: Again, my IDE has made some none requested cleaning, sorry about that. Changes appear here.

**Comment by acolombier** [comment]
Tests are failing because of the missing package update, but as this one is already appearing in the PR#222, I haven't change `package.json` to avoid conflicts.

File: src/validator.ts (hunk 1 above)
Comment (martysweet) [reply src/validator.ts]: Can you clean this up before committing? It makes retrospective diff viewing unpleasant.

**Comment by martysweet** [comment]
Hi @acolombier,

Can you update `package.json` so the tests pass? If you give me permission to the branch I can then resolve any conflicts and merge for release :+1: 

Thanks!
Marty

**Pull request merged**